### PR TITLE
Feat: Home page

### PR DESCRIPTION
**Pull request by sri-rishi**
### Files to check

- HomePage.jsx
- HeroSection.jsx
- NewCategorySection.jsx
- CategorySection.jsx
- BrandCategory.jsx
- InterestCategory.jsx

#### HeroSection

image
 
#### NewCategorySection

image

#### CategorySection

image

#### BrandCategorySection

image

#### InterestCategorySection

image

### CODE REVIEW

File: src/pages/Home/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Footer } from "../../Components/Footer/Footer";
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { BrandNameSection } from "./components/BrandCategory";
+import { CategorySection } from "./components/CategorySection";
+import { HeroSection } from "./components/HeroSection";
+import { InterestCategorySection } from "./components/InterestCategory";
+import { NewCategorySection } from "./components/NewCategorySection";
+
```
Comment (Ankur9669) [inline src/pages/Home/HomePage.jsx:3]: Suggestion: You can import all the files in one single file

**Review by Ankur9669 — APPROVED** [review]
The code looks good to me just a small UI suggestion.

# UI Suggestion
image

The spacing seems abnormal in mobile view

File: src/pages/Home/HomePage.jsx (hunk 1 above)
Comment (sri-rishi) [reply src/pages/Home/HomePage.jsx:3]: I will do that

File: src/pages/Home/components/InterestCategory.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            id: 3, 
+            imageSrc: travelInterestLogoImage,
+            altText: "Travel Interest Category"
+        },
+        {
+            id: 4, 
+            imageSrc: instantInterestLogoImage,
+            altText: "Instant Interest Category"
+        }
+    ]
```
Comment (bhtibrewal) [inline src/pages/Home/components/InterestCategory.jsx]: move this data, to file in data folder

**Review by bhtibrewal — APPROVED** [review]
everything looks good, okay to merge with some minor changes

File: src/pages/Home/components/InterestCategory.jsx (hunk 2 above)
Comment (sri-rishi) [reply src/pages/Home/components/InterestCategory.jsx]: Thank for suggesting , Done

File: src/pages/Home/HomePage.jsx (hunk 1 above)
Comment (sri-rishi) [reply src/pages/Home/HomePage.jsx:3]: done

**Pull request merged**